### ISSUE TITLE
Digital Ocean support IPv6

### ISSUE THREAD

**Issue by Fale** [ansible/ansible-modules-core#2396]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

digital_ocean_domain module
##### ANSIBLE VERSION

N/A
##### SUMMARY

Please, support the ipv6 option while creating new droplet on digital ocean

**Comment by robynbergeron** [comment]
[module: cloud/digital_ocean/digital_ocean_domain.py]

**Comment by robynbergeron** [comment]
@Fale -- Thanks for submitting this feature idea.

@mgregson -- Pinging you to bring this feature request to your attention. Please note that you are under no particular obligation to add this feature.

We encourage all Ansible community members to submit pull requests to satisfy feature requests. Consult the module guidelines (http://docs.ansible.com/ansible/developing_modules.html) for more information.

also pinging @skottler

**Comment by ansibot** [comment]
@mgregson, @alukovenko, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by alukovenko** [comment]
Hopefully will be supported in #4219 
wontfix
